modules/serverless-logic-sending-rest-requests-openAPI-specification.adoc

```diff
@@ -0,0 +1,212 @@
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* You have access to the OpenAPI specification files.
+
+.Procedure
+
+. To define the OpenAPI functions: 
+
+.. Identify and access the OpenAPI specification files for the services you intend to invoke.
+
+.. Copy the OpenAPI specification files into your workflow service directory, such as `src/main/resources/specs`.
++
+Here is an example of the OpenAPI specification for the multiplication REST service:
++
+.Example multiplication REST service OpenAPI specification
+[source,yaml]
+----
+openapi: 3.0.3
+info:
+  title: Generated API
+  version: "1.0"
+paths:
+  /:
+    post:
+      operationId: doOperation
+      parameters:
+        - in: header
+          name: notUsed
+          schema:
+            type: string
+          required: false
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MultiplicationOperation'
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  product:
+                    format: float
+                    type: number
+components:
+  schemas:
+    MultiplicationOperation:
+      type: object
+      properties:
+        leftElement:
+          format: float
+          type: number
+        rightElement:
+          format: float
+          type: number
+----
+
+.. To define functions in the workflow, use the `operationId` from the OpenAPI specification to reference the desired operations in your function definitions.
++
+.Example function definitions in the temperature conversion application
+[source,json]
+----
+{
+   "functions": [
+     {
+       "name": "multiplication",
+       "operation": "specs/multiplication.yaml#doOperation"
+     },
+     {
+       "name": "subtraction",
+       "operation": "specs/subtraction.yaml#doOperation"
+     }
+   ]
+}
+----
+
+.. Ensure that your function definitions reference the correct paths to the OpenAPI files stored in the `src/main/resources/specs` directory.
+
+. To access the defined functions in the workflow states:
+
+.. Define workflow actions to call the function definitions you added. Ensure each action references a function defined earlier.
+
+.. Use the `functionRef` attribute to refer to the specific function by its name. Map the arguments in the `functionRef` using the parameters defined in the OpenAPI specification.
++
+For more information about mapping parameters in the request path instead of request body, you can refer to the following PetStore API example:
```

Comment:
I'm not sure if the "For more information" is needed here as the example is given anyway. Maybe just state "The following example shows..."?